Userland/Libraries/LibGfx/Bitmap.h

```diff
@@ -309,6 +314,14 @@ ALWAYS_INLINE void Bitmap::set_pixel<StorageFormat::RGBA8888>(int x, int y, Colo
     scanline(y)[x] = rgba;
 }
 
+template<>
+ALWAYS_INLINE void Bitmap::set_pixel<StorageFormat::RGBx8888>(int x, int y, Color color)
+{
+    VERIFY(x >= 0);
+    VERIFY(x < width());
+    scanline(y)[x] = color.value();
```

Comment:
The goal of these `set_pixel` template instances is that for a given `Color`, its value gets converted to the internal representation of the bitmap. This means converting the `ARGB32` value into `RGBx8888`, in this case. Currently this directly copies the value into the buffer, which doesn't seem right.